zerver/lib/bugdown/__init__.py

```diff
@@ -1489,6 +1489,74 @@ def __init__(self, parser: Any) -> None:
         super().__init__(parser)
         parser.markdown.tab_length = 4
 
+class AdmonitionProcessor(admonition.AdmonitionProcessor):
+    """ Process !!! admonition blocks to produce colored status messages
+
+        This was modified from the standard AdmonitionProcessor to:
+
+        1. Change the element used from <p> to <blockquote> so that the
+           rendering in clients without additional CSS classes is reasonable.
+        2. Prefix classes added to the admonition blockquote with `adm-` to
```

Comment:
Maybe call the classes `admonition-`?  We generally don't use abbreviations in the Zulip codebase for readability reasons.

Reply:
Certainly.